scripts/sanitycheck

```diff
@@ -802,6 +802,16 @@ class DeviceHandler(Handler):
         except subprocess.CalledProcessError:
             os.write(write_pipe, b'x')  # halt the thread
 
+        if post_script:
+            with subprocess.Popen(post_script, stderr=subprocess.PIPE, stdout=subprocess.PIPE) as proc:
+                try:
+                    (stdout, stderr) = proc.communicate(timeout=30)
+                    logger.debug(stdout.decode())
```

Comment:
I wonder if it'll be clear that the output is from the script here. Some header or similar might be nice.